README.md

```diff
@@ -41,7 +41,7 @@ steps above.
 
 ## Supported versions
 
-The bindings require dav1d 1.3.0
+The bindings require dav1d 1.4.0
```

Comment:
You can keep this at 1.3.0. It's backwards compatible

Reply:
Indeed, thanks! I have explicited the lower and upper bound so that it's not misleading.